ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml

```diff
@@ -53,26 +53,23 @@ resources:
       memory: 200Mi
 tag_specification:
   cluster: https://api.ci.openshift.org
-  name: "4.5"
+  name: "4.6"
```

Comment:
fyi: at least 2 days ago the 4.6 didn't work yet, we might need to keep 4.5 for now (did the same on the operator repo)